preprocessing/shift_generation.py

```diff
@@ -21,13 +22,18 @@
 def load_data(problem_name):
     root = xml_loader.get_root(problem_name)
 
-    competencies = get_competencies(root)
+    competencies = []
+    #Do not think we will use this anymore
+    #competencies = get_competencies(root)
+    
     staff = get_employee_lists(root, competencies)
-    time_sets = get_time_sets(root)
+    time_sets = get_time_sets(root, competencies)
```

Comment:
Do you update `competencies` somewhere? Or are you just sending in an empty list to these functions?

Reply:
Competencies are updated directly in the get_staff function. Since python only pass a reference I passed the competency set which is empty and adds competencies to the set as they are added to each employee. Do not return it, but when that function is done the passed set is filled